[ADD] auto_emaile_send: send email after n days

haahi-odoo opened the pull request:
-Added functionality in invoicing to send emails automatic
-Added functionality in invoicing settings

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/tutorials/pull/1046.png)](https://mergebot.odoo.com/odoo/tutorials/pull/1046)

haahi-odoo committed:
[ADD] auto_emaile_send: send email after n days

-Added functionality in invoicing to send emails automatic
-Added functionality in invoicing settings
-Added cron job to configure action

bit-odoo commented:
Hello,
Please change the Pull request version.
